src/ui/public/field_editor/__tests__/field_editor.js

```diff
@@ -16,13 +16,13 @@ describe('FieldEditor directive', function () {
   let $el;
 
   let $httpBackend;
+  let getScriptedLangsResponse;
 
   beforeEach(ngMock.module('kibana'));
   beforeEach(ngMock.inject(function ($compile, $injector, Private) {
     $httpBackend = $injector.get('$httpBackend');
-    $httpBackend
-    .when('GET', '/api/kibana/scripts/languages')
-    .respond(['expression', 'painless']);
+    getScriptedLangsResponse = $httpBackend.when('GET', '/api/kibana/scripts/languages');
+    getScriptedLangsResponse.respond(['expression', 'painless']);
```

Comment:
My preference in `beforeEach`: if it gets overridden in any test, it should always be handled within _each_ test (so I only `beforeEach` something if it concerns _all_ the tests it covers, otherwise I find it just becomes more difficult to track the state of things)